apps/www/_blog/2020-10-30-improved-dx.mdx

```diff
@@ -12,7 +12,12 @@ tags:
 date: '10-30-2020'
 ---
 
-Today we're releasing [supabase-js](https://github.com/supabase/supabase-js) version 1.0, and it comes with some major Developer Experience improvements.
+**UPDATE 16/08/2022**: supabase-js v2 is out and focuses on “quality-of-life” improvements for developers. V2 includes Type support, new auth methods, multiplayer sneak peek, and more: [**Read the blog post**](https://supabase.com/blog/supabase-js-v2)
```

Comment:
@RamiroND can we leave out multiplayer for now?

Reply:
@RamiroND or can we call it new Realtime multiplayer sneak peak?

Reply:
Ok done, I've called it realtime multiplayer...